modules/cli/src/integrationTest/java/org/apache/ignite/internal/cli/core/repl/executor/ItIgnitePicocliCommandsTest.java

```diff
@@ -265,7 +265,9 @@ void nodeConfigUpdateSuggested(ParsedLine givenParsedLine) {
         // wait for lazy init of node config completer
         await("For given parsed words: " + givenParsedLine.words()).until(
                 () -> complete(givenParsedLine),
-                containsInAnyOrder("rest", "clientConnector", "network", "cluster", "deployment", "nodeAttributes")
+                containsInAnyOrder(
+                        "rest", "clientConnector", "network", "cluster", "deployment", "nodeAttributes", "aimem", "aipersist", "rocksDb"
```

Comment:
I believe we should use lower case for "rocksdb", but let's discuss it in the future